test_project/store/src/main/java/test/sergeyvalyushko/store/category/Backery.java

```diff
@@ -0,0 +1,30 @@
+package test.sergeyvalyushko.store.category;
+
+import test.sergeyvalyushko.store.Category;
+import test.sergeyvalyushko.store.Product;
+
+import java.util.*;
+
+public class Backery extends Category {
+    private static final String name = "Backery";
+    @Override
+    public String getName(){
+        return name;
+    }
+
+    @Override
+    protected List<Product> addProduct() {
+        productList.add(new Product("loaf", 5.1, new Date()));
```

Comment:
please reread 3d task from new program, https://coherentsolutions.sharepoint.com/sites/training-center/Programs/Forms/AllItems.aspx?id=%2Fsites%2Ftraining%2Dcenter%2FPrograms%2FJava%20for%20automation%2F03%5Foop%2Emd&parent=%2Fsites%2Ftraining%2Dcenter%2FPrograms%2FJava%20for%20automation, and add faker functionality